lib/ansible/modules/windows/win_wait_for_process.ps1

```diff
@@ -1,59 +1,62 @@
 #!powershell
-# This file is part of Ansible
 
-# Copyright (c) 2017 Ansible Project
+# Copyright: (c) 2017, Ansible Project
+# Copyright: (c) 2018, Dag Wieers (@dagwieers) <dag@wieers.com>
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 #Requires -Module Ansible.ModuleUtils.Legacy
-#Requires -Module Ansible.ModuleUtils.FileUtil
+#Requires -Module Ansible.ModuleUtils.SID
 
 $ErrorActionPreference = "Stop"
 
+trap {
+    $result.exception = "$($_ | Out-String)`r`n$($_.ScriptStackTrace)"
```

Comment:
Maybe add a comment as to why the trap is here.

Reply:
Should we keep it in ?

Reply:
up to you, I haven't fully thought out what some of the consequences could be but I was ultimately planning on having the exec_wrapper inject something like this into the module on execution to give us a prettier output.

Reply:
Yes please ! I think it will help us with some behaviour.
The sooner the better if you ask me ;-)

Reply:
Added note !